interface-elements-for-desktop/articles/report-designer/report-designer-for-winforms/shape-report-data/use-report-parameters/parameters-overview.md

```diff
@@ -4,75 +4,95 @@ author: Anna Gubareva
-* **Default value** - specifies the default parameter value.
-* **Show in the parameters panel** (corresponds to the parameter's **Visible** property) - enable this option to request the parameter value in a Print Preview. Otherwise, the default parameter value is silently passed to the report.
-* **Supports the collection of standard values** - you can enable this option if the parameter is visible (i.e., its value should be requested in a Print Preview). In this case, you can choose a value from a predefined list. You can either manually populate this list with possible values, or specify a data source from where these values should be obtained.
-	
 	* **Dynamic values**
-		
-		On this tab, you can specify a data source, data adapter (if required) and data member storing parameter values. The value member defines a data field that will provide values to the parameter. The display member defines a data field storing values displayed in a Print Preview.
+
+		Specify a data source, data adapter and data member for the parameter values storage. **Value Member** defines a data field that provides values to the parameter. **Display Member** defines a data field that stores values displayed in Print Preview.
 		
 		![](../../../../../images/eurd-win-parameters-dynamic-values.png)
 		
-		The value type of the specified data member should match the specified parameter type.
+		The specified data member's value type should match the specified parameter type.
 		
-		You can filter the list of values by specifying the **Filter String** property. Using this property, you can implement [cascading parameters](create-multi-value-and-cascading-parameters.md).
+		Use the **Filter String** property to filter parameter values and implement [cascading parameters](create-multi-value-and-cascading-parameters.md).
 
-		You can sort the list of values by specifying the **Sort Member** and **Sort Order** properties.
+		Specify the **Sort Member** and **Sort Order** properties to sort parameter values.
 		
 	* **Static values**
 		
-		Switch to this tab to specify a static list of possible values. Each value should have a description that is displayed in a Print Preview.
+		Switch to this tab to specify a static list of values. Each value should have a description that is displayed in Print Preview.
 		
 		![](../../../../../images/eurd-win-parameters-static-values.png)
 
-* **Allow multiple values** (corresponds to the parameter's **Multi-Value** property) - when this option is enabled, a parameter can be assigned a [collection of values](create-multi-value-and-cascading-parameters.md).
+## Use Parameters
 
-* **Allow null value** (corresponds to the parameter's **Allow Null** property) - indicates whether the parameter's value can be unspecified.
+Report parameters pass data to a report before it is generated in Print Preview. You can specify parameter values.
```

Comment:
"You can specify parameter value" This is out of context.

Reply:
Changed